css/main.css

```diff
@@ -124,8 +126,55 @@
     transition: all 0.5s ease;
     cursor: pointer;
   }
-  .hamburger::before { top: 6px; }
-  .hamburger::after { bottom: 6px; }
+
+  .hamburger::before {
+    top: 6px;
+  }
+
+  .hamburger::after {
+    bottom: 6px;
+  }
+
+  .text {
+    width: 327px;
+    margin: 1.8rem 2rem 0 1.2rem;
+  }
+
+  .work-section .work-header {
+    width: 327px;
+    line-height: 75px;
+  }
+
+  .work-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 68px;
+  }
+
+  .about-text img {
+    top: 50px;
```

Comment:
- [ ] KIndly make sure your webpage is not overflowing on the right side as you can see. We don't need to scroll to the right to see the entire content of the page. as you can see here :point_down: 
![Screenshot from 2022-09-23 01-06-27](https://user-images.githubusercontent.com/97100091/191866794-42af1f7d-37a0-4693-837b-9d38fbc22b69.png)

- [ ] KIndly make sure the `typo` and `font` are met with the ones provided in the Figma template

Reply:
Using overflow property will make the page stuck. As for the picture the media query used on it is way far from responsive devices @christianbiring1

Reply:
Typo and fonts is all correct @christianbiring1